build/unix/module.modulemap

```diff
@@ -8,6 +8,11 @@ module ROOT_Foundation_C  {
   export *
 }
 
+module ROOT_FwdNamespaces {
+  module "input_line_fwd.h" { header "input_line_fwd.h" export *}
```

Comment:
We should probably change the name to something like: `AutoGeneratedForwardDeclaredNamespacesForGMI.h`